### ISSUE TITLE
What happened to this?  was this project abandoned? =(

### ISSUE THREAD

**Issue by pabloacera** [r-bioinformatics/edgePy#81]

**Comment by apfejes** [comment]
Yup - I changed jobs and no one else wanted to pick it up.  Sorry for the bad news.

**Comment by sonti-roy** [comment]
Hi @apfejes  
Are you interested in continuing the project? or Is there any possibility of doing that?
I would like to be a contributor.

**Comment by uamoti** [comment]
I'd also like to contribute.
I don't have much experience but I'm looking to move into dev work and this project would definitely be a great step in that direction.

**Comment by apfejes** [comment]
Truthfully, I really don't have the time to contribute to this project, but there are likely others who would be interested in mentoring or helping make this happen.  I suggest joining the reddit r/bioinformatics slack group, and we can see if any of the original contributors would be willing to take on that role.

**Comment by julistanley** [comment]
As mentioned in the slack group the other day, folks who are interested might also consider contributing to PyDESeq2.

[Repo](https://github.com/owkin/PyDESeq2) | [Preprint](https://www.biorxiv.org/content/10.1101/2022.12.14.520412v1)

**Comment by nvannest** [comment]
As far as I can tell, PyDESeq2 does not give users the ability to perform TMM which is what EdgePy was useful for--Did anyone find a good alternative?
